scripts/build-osx-app.sh

```diff
@@ -139,11 +141,34 @@ sed -i.bak "s@/.*\.app/@$TEMPLATE/@g" "${SITE_PACKAGES}"/sipconfig.py
     "$PIP" install qt-graph-helpers
 )
 
+# Explicitly specify a numpy version due to an undeclared dependency of
+# scikit-learn's osx wheel files (0.17.1) which seem to be build against
+# numpy 1.10.*, and require numpy >= 1.10 ABI.
+echo "Installing numpy"
+echo "================"
+
+"$PIP" install --only-binary numpy 'numpy==1.10.*'
```

Comment:
As we have already released and published dmg with numpy 1.10, I will merge this as-is.